app/assess/templates/project_summary.html

```diff
@@ -20,18 +21,39 @@ <h1 class="govuk-heading-l">{{application_data.identifier}}</h1>
         'text': fund_data.description,
         'id': "assessment-page-details",
         'open':True}) }}
-<p class="govuk-body-l">You have completed X of Y questions.</p>
-
-{{ govukTable({'caption': 'General assessment',
-'captionClasses': 'govuk-heading-l',
-'rows': [[{'text': '_______'},
-{'text': '________', 'format': 'numeric'}],
-
-[{'text': '_______'}, {'text': '________', 'format': 'numeric'}],
-
-[{'text': '_______'},
-{'text': '________', 'format': 'numeric'}]]}) }}
 
 
+<p class="govuk-body-l">
+   You have completed {{status_COMPLETED|length}}
+   of {{status_NOT_COMPLETED|length}} questions.
```

Comment:
If status_NOT_COMPLETED is just used to get the length why not just pass in a variable called "num_of_completed_questions"? You'd just need to run Len on your list comprehensions.

Reply:
Also check out https://design-system.service.gov.uk/patterns/task-list-pages/ for some context if you have time.

Reply:
Yes, that would make more sense! Done for both!

   number_of_questions_not_completed =  {
        question: status
        for question, status in statuses.items()
        if (status == "NOT STARTED" or status == "IN PROGRESS")
    }

    return len(number_of_questions_not_completed)